pkgs/top-level/python-packages.nix

```diff
@@ -6096,6 +6096,8 @@ in {
 
   pdunehd = callPackage ../development/python-modules/pdunehd { };
 
+  pixcat = callPackage ../development/python-modules/pixcat { };
```

Comment:
If the package is not used as a python depdency it shouldn't be added here.

Reply:
The package is meant as a python dependency as well, is that enough?